### DISCUSSION TITLE
Is df.to_records() available in vaex 4.3.0 version?

### DISCUSSION THREAD

**Discussion by bsbalkar** [vaexio/vaex#1489] [Q&A]
I'm having vaex 4.3.0 on my windows machine. As per the online documentation df.to_records() is available in vaex 4.3.0.
But it's not in the vaex module installed in my windows machine.
Will this function be available in vaex 4.4.0?

**Comment by JovanVeljanoski** [comment]
The docs I believe are tracking the current master branch. 
So everything now in master will be part of the next release.

**Comment by vivianfang88** [comment]
Hello, I am using Vaex version 4.3.0.
I have encountered error message "'DataFrameLocal' object has no attribute 'to_records'". It seems like  df.to_records() is not in the Vaex module. May I know why I am getting this error?
